api/health.go

```diff
@@ -12,6 +12,20 @@ import (
 
 // Health represents the API handler to
 // report the health status for Vela.
+// swagger:operation GET /health router Health
```

Comment:
Thoughts on moving the function description comments below the swagger specific comments?

My thought process is that it might help (or be nice) to remain consistent 👍

Reply:
To clarify - do something similar to login.go?
https://github.com/go-vela/server/blob/4a8063d440152c6a580ff9466a68bd400489bc9d/api/login.go#L77-L78

Reply:
@meinenec Yes that is correct 👍